fms_core/resources/sample.py

```diff
@@ -29,7 +29,7 @@
 
 class SampleResource(GenericResource):
     # Simple model fields
-    biospecimen_type = Field(attribute='biospecimen_type', column_name='Biospecimen Type')
+    sample_kind = Field(attribute='sample_kind_name', column_name='Sample Kind')
```

Comment:
Same as earlier... you don't need to invent an attribute if you are not using the automated mapping.